packages/beachball/src/changefile/promptForChange.ts

```diff
@@ -91,3 +94,18 @@ export async function promptForChange(options: BeachballOptions) {
 
   return packageChangeInfo;
 }
+
+function getDisallowedChangeTypes(
```

Comment:
It seems like this is a good start - where the group disallowedChangeTypes will override the setting for the individual packages.